README.md

```diff
@@ -79,21 +88,123 @@ POINTS
 1 0 0
 
 ```
+Parameters could be passed as
+* keyword arguments (as above),
+* `Pair{Symbol, ...}`s e.g. `Polytope.Polytope(:POINTS=>[ ... ])`
+* dictionaries e.g. `Polytope.Polytope(Dict( "POINTS" => [ ... ] )`)
 
-Properties of such objects can be accessed by the `.` syntax
+The dictionary may hold many different attributes. All the names *must* be compatible with `perl/polymake`.
+
+Properties of such objects can be accessed by the `.` syntax:
 ```
 julia> p.INTERIOR_LATTICE_POINTS
 pm::Matrix<pm::Integer>
 1 0 0
 ```
 
+### Example script
+
+The following script is modelled on the one from the [Using Perl within polymake](https://polymake.org/doku.php/user_guide/tutorials/perl_intro) tutorial:
+
+```julia
+using Polymake
+
+str = read("points.demo", String)
+matrix_str = "["*replace(str, "/"=>"//")*"]"
+matrix = eval(Base.Meta.parse(matrix_str))
+@show matrix
+
+p = @pm Polytope.Polytope(:POINTS=>matrix)
```

Comment:
I think we should aim for a favourite constructor (since we support all kind of variants).
I like `@pm Polytope.Polytope(POINTS = matrix)` more since we don't need to explain symbols and it is more consistent with kwargs constructors.

Reply:
I'm in favour of `kwargs` convention as well